.github/workflows/dockerhub-release.yml

```diff
@@ -0,0 +1,51 @@
+name: Dockerhub Release Pipeline
+
+on:
+  release:
+    types: [created]
+
+env:
+  ENV: "prd"
```

Comment:
On the app config we're calling the prod environment "prod". Perhaps it would make sense to name it the same way?

Reply:
it makes sense, but it is not directly used, not sure if would have any impact on the build process. I think it should be removed anyway

Reply:
We would have some work to do on the mdw in regards to prod config and releases. It's in the scope of #195